sumologic-service/0.15.0/README.md

```diff
@@ -0,0 +1,174 @@
+If you want to learn more about Keptn visit us on [keptn.sh](https://keptn.sh)
+
+Check the issue here for more info: https://github.com/keptn/integrations/issues/20
+
+## Quickstart
+If you are on Mac or Linux, you can use [examples/kup.sh](https://raw.githubusercontent.com/keptn-sandbox/sumologic-service/release-0.15.0/examples/kup.sh) to set up a local Keptn installation that uses Sumo Logic. This script creates a local minikube cluster, installs Keptn, Istio, Sumo Logic and the Sumo Logic integration for Keptn (check the script for pre-requisites). 
+
+To use the script,
+```bash
+export ACCESS_ID="<your-sumologic-access-id>" ACCESS_KEY="<your-sumologic-access-key>"
+examples/kup.sh
+```
+Check [the official docs](https://help.sumologic.com/Manage/Security/Access-Keys#manage-your-access-keys-on-preferences-page) for how to create the Sumo Logic access ID and access key
+
+## If you already have a Keptn cluster running
+**1. Install Sumo Logic**
+```bash
+export ACCESS_ID="<your-sumologic-access-id>" ACCESS_KEY="<your-sumologic-access-key>"
+helm upgrade --install my-sumo sumologic/sumologic   --set sumologic.accessId="${ACCESS_ID}"   --set sumologic.accessKey="${ACCESS_KEY}"   --set sumologic.clusterName="keptn-sumo"
+
+```
+**2. Install Keptn sumologic-service to integrate Sumo Logic with Keptn**
+```bash
+export ACCESS_ID="<your-sumologic-access-id>" ACCESS_KEY="<your-sumologic-access-key>"
+# cd sumologic-service
+helm install sumologic-service ../helm --set sumologicservice.accessId=${ACCESS_ID} --set sumologicservice.accessKey=${ACCESS_KEY} 
+
+```
+
+**3. Add SLI and SLO**
+```bash
+keptn add-resource --project="<your-project>" --stage="<stage-name>" --service="<service-name>" --resource=/path-to/your/sli-file.yaml --resourceUri=sumologic/sli.yaml
+keptn add-resource --project="<your-project>"  --stage="<stage-name>" --service="<service-name>" --resource=/path-to/your/slo-file.yaml --resourceUri=slo.yaml
+```
+Example:
+```bash
+keptn add-resource --project="podtatohead" --stage="hardening" --service="helloservice" --resource=./quickstart/sli.yaml --resourceUri=sumologic/sli.yaml
+keptn add-resource --project="podtatohead" --stage="hardening" --service="helloservice" --resource=./quickstart/slo.yaml --resourceUri=slo.yaml
+```
+Check [./quickstart/sli.yaml](https://raw.githubusercontent.com/keptn-sandbox/sumologic-service/release-0.15.0/examples/quickstart/sli.yaml) and [./quickstart/slo.yaml](https://raw.githubusercontent.com/keptn-sandbox/sumologic-service/release-0.15.0/examples/quickstart/slo.yaml) for example SLI and SLO. 
+
+<!-- TODO: Uncomment this after the PR to support switching SLI provider is merged -->
```

Comment:
Maybe we should wait on this PR before merging the documentation. Also, will this be added to older Keptn versions like 0.15.0 or only to newer ones?

Reply:
> Maybe we should wait on this PR before merging the documentation.

I did think of this. I have added a paragraph around [how to get around this](https://github.com/keptn-contrib/artifacthub/pull/148/files#diff-e90c6b2e1e5b1faa9db5ffe910589d757297d801f76a00562af89bf4f2aec1dcR53-R64) in the current README. I think the CLI shouldn't block us from releasing this integration because it will take some time before the CLI PR gets merged and we test this integration against the newer Keptn version which has the CLI PR.

 > Also, will this be added to older Keptn versions like 0.15.0 or only to newer ones?

No, only the new ones. The CLI is a convenience for not doing [this](https://github.com/keptn-contrib/artifacthub/pull/148/files#diff-e90c6b2e1e5b1faa9db5ffe910589d757297d801f76a00562af89bf4f2aec1dcR53-R64) manually.

Reply:
Putting this in artifactory would give this integration more visibility and we can get more users for trying it out (since the integration is still experimental).